scripts/test-all.js

```diff
@@ -107,11 +109,21 @@ const main = async () => {
     const references = loadReferences('tsconfig.json');
     const start = Date.now();
 
-    console.log(chalk.green.bold(`Testing all packages`));
+    console.log(chalk.green.bold(`${action} all packages`));
```

Comment:
Good change, should we rename this file to something like `build-or-test-all.js` or `run-all.js` while we're at it?

Reply:
sgtm